client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -132,10 +141,16 @@ export default connect(
 			true !== getSiteOption( state, siteId, 'publicize_permanently_disabled' ) &&
 			postTypeSupports( state, siteId, postType, 'publicize' )
 		);
+		const isSharingActive = false !== isJetpackModuleActive( state, siteId, 'sharedaddy' );
+		const isLikesActive = false !== isJetpackModuleActive( state, siteId, 'likes' );
+		const isJetpack = false !== isJetpackSite( state, siteId );
 
 		return {
+			connections: getSiteUserConnections( state, siteId, userId ),
+			isSharingAndLikesDisabled: ! isSharingActive && ! isLikesActive,
+			isJetpack,
```

Comment:
Do we need this prop, since `isJetpackModuleActive` is already checking for the site to be Jetpack and will never be explicitly `false` for a WordPress.com site?

From docs:

>Returns null if the site is not known or is not a Jetpack site.